---
published: true
post_snippet: ...identified 34 gut-microbiota associated epitopes (MEs) and demonstrated that these were abnormal in the gut of ASD children in terms of composition and was in turn associated with abnormal IgA levels.
title: Gut microbiota-associated epitopes in children with autism spectrum disorders
---

#### [Wang et al. In Press. Alteration of gut microbiota-associated epitopes in children with autism spectrum disorders. Brain, Behavior, and Immunity.](https://www.sciencedirect.com/science/article/pii/S0889159118307414?via%3Dihub)

_Summarized from DalMUG group discussion and written by:
**Alessi Kwawukume**_

### Summary
The rationale of this study was to determine if the gut-microbiota was associated with the pathogenesis of autism spectrum disorder (ASD), through immune-mediated pathways. The study identified 34 gut-microbiota associated epitopes (MEs) and demonstrated that these were abnormal in the gut of ASD children in terms of composition and was in turn associated with abnormal IgA levels. The take home message was that the MEs identified separated ASD children from typically developing (TD) children and may therefore serve as potential biomarkers for ASD. Overall the methods used were straightforward, however during the discussion, the group believed more work could have been put in the methods and discussion to draw a more convincing conclusion.

### Points of Interest
* It was interesting to observe that most of the differences between ASD and TD children were between ASD children with Gastrointestinal (GI) problems. It seemed the differences were more as a result of GI problem than ASD. It would have been a good addition if TD children with GI problems were also recruited for the study.
* Also, of interest was why the IgA levels of Children with ASD were higher than those of TD children while the number of MEs derived from proteins of pathogenic microorganisms were higher in TD children than ASD children.
* Though the MEs in ASD children with or without GI problems were comparable, it would have been nice to have included the severity of ASD and see how that compares in terms of ME composition and IgA levels

### Points of Confusion
* The method for determining the MEs were not quite clear. We were not familiar with the IEDB database, but the group believed that BLASTing metagenomic reads against the database was not the best option to predict MEs
* The differences observed between MEs in the 3 groups studied (ASD without GI, ASD with GI and TD children) did not look obvious in the PC graph (Figure 3)
* It would also have been good to analyze IgA levels with respect to the MEs identified
