---
published: true
title: Bacterial exclusivity offers new taxonomic insights
post_snippet: Do bacterial species exist other than as a functional delineation? Exclusivity says no. 
---
[Wright & Baum. 2018.Exclusivity offers a sound yet practical species criterion for bacteria despite abundant gene flow. BMC Genomics.] (https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-5099-6)

_This is a summary of our DalMUG journal club discussion of the above paper written by **Emily Lamoureux**._

### Summary

Big research question: with so much HGT, do discrete bacterial species really exist? In short – no but we can still delineate taxa in a way that is useful for science. Built a tree ignoring the historical groupings of bacteria/evolutionary lines and focused only on genes. Both the core-genome and pan-genome of Streptomyces are used to demonstrate exclusivity. Exclusivity: more related to each other than organisms outside the group. This works for average relatedness across the genome. 

### Points of Interest

* Figure 1 - 16S has poor correlation with gene similarity which has interesting implications for PICRUSt. However, PICRUSt only uses annotated stuff from databases 
* Fundamental – is there a delineation at species other than a practical application? Taxonomy is just practical – no biological/evolutionary drive. How do we apply our functional knowledge? Could slowly switch over to classification based on function and do the conversion (update our taxa to be more accurate). 
* Didn’t oversell their conclusions in the discussion. Didn’t try to make up a good cutoff for species but fit their idea into existing methods. Stated that naming scheme is arbitrary so they used traditional nomenclature
* Useful method for microbial epidemiology, where you are using the evolutionary tree but are interested in gene acquisition 

### Points of Confusion

* Figure 1: gap at 97% similarity in the ANI panel (bottom left). Possibly biased based on assemblies that we have put into databases. Can you get around it by subsampling? 
* They argue that exclusivity is a better metric for defining species even though there is no exact cutoff. ANI people have argued this. We have some that work to some degree. What we’re doing by applying exclusivity to ANI levels is working already
* Their metric breaks down/changes over time as you add more taxonomic information, whereas ANI is static

