---
title: Extensive unexplored human microbiome diversity
post_snippet: Many microbes in the human microbiome remain unknown. This issue is especially pervasive for individuals on non-Western diets since they are underrepresented in microbiome studies.
---

[Pasolli et al. 2019. Extensive Unexplored Human Microbiome Diversity Revealed by Over 150,000 Genomes from Metagenomes Spanning Age, Geography, and Lifestyle. Cell.](https://www.sciencedirect.com/science/article/pii/S0092867419300017?via%3Dihub)

This is a summary of our DalMUG journal club discussion of this paper written by _Jocelyn MacDonald_.

### Summary
Many microbes in the human microbiome remain unknown. This issue is especially pervasive for individuals on non-Western diets since
they are underrepresented in microbiome studies.  To better understand the unknown microbial variation across populations,
shotgun metagenomics samples were acquired from 46 public datasets, and collected from a novel cohort in Madagascar.
These samples came from the stool, skin, oral, vagina, and breast milk, and the studied individuals varied in age and geographic
location. Pasolli et al. used large scale metagenomic assembly of these metagenomes to identify previously unidentified and
uncharacterized microbes.

### Points of Interest
* Interesting to see the difference in tryptophan metabolism between westernized and non-westernized populations
* Mappability of the gut increased by 87% when using these reconstructed genomes
* 4,930 species were found, but only 23% of these have been identified before


### Points of Confusion
* How could these genomes be used now? Should they be incorporated into existing tools (i.e. are they of sufficient quality)?
* It was unclear what the separation between western vs non-western was. Is it based on geographic location? Or consumption of the western diet? The paper was lacking a clear definition of the term.
* Since many of these taxa have never been identified before except through amplicon sequencing, how accurate are the assembly quality metrics?

### Author Feedback
The lead author Edoardo Pasolli gave some nice feedback on our points of confusion above. See the Twitter thread here:

<ul>
<li>
<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Thanks for the nice summary! Just to clarify your points: 1) the genomes could be used for comparative genomics (figs. 3-4-5 are sort of examples) and for integrating databases for mapping-based approaches</p>&mdash; Edoardo Pasolli (@epasolli) <a href="https://twitter.com/epasolli/status/1112968451977633793?ref_src=twsrc%5Etfw">April 2, 2019</a></blockquote>
<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
</li>
  
