---
published: true
title: Resident Enterobacteriaceae in healthy human adults
post_snippet: ... longitudinal human gut microbiome datasets of healthy adult to identify resident and transient members of microbiome.
---

#### [Martinson et al., 2019. "Rethinking gut microbiome residency and the _Enterobacteriaceae_ in healthy human adults." ISME J](https://www.nature.com/articles/s41396-019-0435-7)

_Summarized from DalMUG group discussion and written by: **Akhilesh S. Dhanani**_

### Summary
The dynamics of the human gut microbiome have been studied in relation to a variety of human diseases. However, the information on fundamental aspects of microbiome such as the residency time of microbes, colonization and persistence of some microbes over others in human gut, remains sparse. Again, longitudinal human gut microbiome studies are limited and contrasts the observation of stable subset of long-lived resident microbes between culture-and sequence-based approach. Martinson et al. attempted to identify static and dynamic members of healthy human gut microbiome in a small longitudinal cohort of eight adult at the cultured clone level and operational taxonomic units (OTUs)/amplified sequence variants (ASVs) levels based on 16S rRNA sequencing. Biweekly sample collection over a two year span yielded 390 samples in total for culture-based evaluation of _Enterobacteriaceae_, of which 324 were also analysed by 16S rRNA sequencing.

Processing the sequenced reads of the 16S rRNA gene (V4 region) resulted in 293 OTUs and 453 ASVs. Non-metric multidimensional scaling (NMDS) grouped samples belonging to the same participants and partitioning around mediods (PAM) clustering showed 7 and 8 statistically significant clusters for OTUs and ASVs, respectively. The change in beta diversity from one sampling point to the next was small and all participants (except 4 and 5) showed similar beta diversity. Although there was a significant positive correlation of beta diversity increased with time, the small correlation coefficients and the effect size of beta diversity change suggests the microbiome profiles sampled around the same time are similar to those sampled over long periods of time. Based on residency time (same OTUs/ASVs found over 14 days), more than 30% OTUs/ASVs were resident for the entire study period. Additionally, the percentile residency rank based on OTUs and ASVs suggested that a significant proportion (~50%) of the microbiome was rarely resident, although each participant maintained a unique profiles throughout all time-points.

Out of 32,470 _Enterobacteriaceae_ colonies screened with PCR-based assays, 28,156 (86%) belonged to one of ten _E. coli sensu stricto_ phylogroups. Furthermore, GTG5 rep-PCR identified total of 120 distinct _Enterobacteriaceae_ clones among all participants and 37 (31%) of these were resident (including 3 non- _E. coli_).


### Points of Interest
- As longitudinal studies are limited, this study adds more insight to available information on dynamics of microbiome as well as short and long-lived members of the healthy human gut microbiome. The overall approach to bin OTUs/ASVs into resident and transient taxa based on residency period is interesting.
- The sequence based analysis with OTUs/ASVs showed similar microbiome within participants over long periods of time, although there was substantial flux of transient members.
- The culture-based approach to identify Enterobacteriaceae lead to a significant variety of clones compared to the sequence-based approach. Interesting to note that most of the identified _Enterobacteriaceae_ predominantly belong to _E. coli_ phylotypes and many of them were not lactose fermenters. 

### Points of Confusion
- Although efforts were made to select unique colonies from MacConkey agar plates with colony morphology, only one plate from each sample with dilution were used. This could lead to the selection of the most abundant and redundant clones rather than covering vast diversity. As it appears from the large pool of _Enterobacteriaceae_, only 120 distinct clones were found.
- Using a similar sequence-based approach on cultured isolates would have added weight to the conclusion that the 16S sequencing and culture results did not correlate well. In addition, sequencing the 16S rRNGA gene V4 region provides limited information for resolving strain-level variation. There was no abundance/count mentioned for OTUs and ASVs corresponding to _Enterobacteriaceae/E. coli_.
- The sequence-based technique is limited in general as a diagnostic tool and is known to have little power to discriminate taxa with high degrees of variation at the strain level (like _Enterobacteriaceae_).
