---
title: Strain adaptation in the early life human microbiome
post_snippet: Variation at the bacterial strain level sheds light on functional differences amongst infant gut microbiomes from Eastern Europe. 
---

[Vatanen et al., 2019. Genomic variation and strain-specific functional adaptation in the human gut microbiome during early life, Nature Microbiology.](https://www.nature.com/articles/s41564-018-0321-5)

_This is a summary of our DalMUG journal club discussion of this paper written by **[Casey Jones](https://twitter.com/CaseyMAJones)**_

### Summary
Tommi Vatanen and colleagues describe a well-sampled cohort of children from the [DIABIMMUNE study](https://pubs.broadinstitute.org/diabimmune/three-country-cohort) that consists of 289 children who carry HLA haplotypes that confer an increased risk of autoimmune disorders. The researchers collected a longitudinal set of over 3000 stool samples through following the subjects from birth for three years from children in Finland, Estonia, and Russian Karelia.  

In this publication, the authors focus on microbial strain diversity through two methods: single nucleotide polymorphism (SNP) tracking (using StrainPhlAn), and assembly-based strain tracking. Due to the longitudinal nature of the study, they were able to track genetic changes in microbes within an infant over time. Microbes that had the greatest genetic diversity included *Haemophilus parainfluenzae* and *Faecalibacterium prausnitzii*. 

Through CRISPR array reconstruction, they also suggest that bacteriophages may be contributing to the observed genomic changes in *Bacteroides* spp., although more in-depth work is needed to confirm this mechanism. 

We were satisfied overall with this publication that describes a cohort that will better delineate the link between early human gut microbiome development and host immunity. 

### Points of Interest
* Bacteria usually found in the oral cavity in adults spiked in the first year of life in some infants
* Early growth, urban/rural household status, and antibiotic courses during pregnancy were associated with microbial composition. Children living outside cities had a more diverse microbiome 
* *Bifidobacterium infantis* was shown to be enriched in samples collected during breastfeeding, a functional consequence of their ability to metabolize and transport human milk oligosaccharides (HMOs)
* Potential evidence for engraftment of *Bifidobacterium bifidum* probiotic in Russian infants 
* Mostly one species (*E. coli*) contributes to siderophore biosynthesis (iron-binding compounds) while siderophore-transport associated genes are widely distributed amongst several species 


### Points of Confusion
* Evidence for phages targeting *Bacteroides* spp. needs further investigation
* Timescale in Figure 3a & 3b could be flipped 
