---
published: true
title: The human tumor microbiome is composed of tumor type–specific intracellular bacteria

post_snippet: ...completed a comprehensive 16S rRNA gene survey of the microbiome of various tumor types (breast, lung, ovary, pancreas, melanoma, bone and brain).
---

[Nejman et al. 2020. The human tumor microbiome is composed of tumor type–specific intracellular bacteria. Science.](https://science.sciencemag.org/content/368/6494/973)

_This is a summary of our DalMUG journal club discussion of this paper written by Chris Tang_


### Summary
Nejman et al. completed a comprehensive 16S rRNA gene survey of the microbiome of various tumor types (breast, lung, ovary, pancreas, melanoma, bone and brain). In this multi-center study, the authors screened over 2000 tumors, healthy adjacent tissue and tissue from healthy controls for the presence of bacteria. By examining these samples they determined that intracellular bacteria are present within tumor tissue and tumor-associated immune cells using a variety of techniques combining both culturomics and microscopy. The authors identified over 500 different bacterial species across these seven tumor types by using a novel multi-amplicon 16S rRNA sequencing technique coupled with a rigorous filtering strategy to remove possible contaminants. Bacterial composition at the genus and species levels varied across tumor types and their inferred functions were found to be associated with various tumor types and subtypes. Overall, we thought this was a rigorous study though additional validation would be beneficial to establish that the tumor microbiome is measurably different compared to the microbiome of normal tissue.

Below are a few key points of interest and confusion that came up during our discussion.


### Points of interest
- Fig. 4B: It was interesting to that Streptophyta (i.e. likely plant chloroplasts) was identified in tumors.
- Nice to see that predicted functions that were differentially abundant in lung and breast tumor subtypes seemed biologically plausible

### Points of confusion
- Fig. 3: Is there a rarefaction curve based on sequencing depth available?
- Fig. 4B: Many species were filtered out using filtering techniques that have not been previously benchmarked.
- Fig. 4D: It was unclear if the uncoloured circles represented species that were filtered out.
- Fig. 4F: It was unclear why the PCoA plot was used to summarize tissue types rather than just plotting the samples themselves. It seemed like there may have been too much variation within individual samples to get sensible distance measures.
- Fig. 5: Were there any other comparisons done between other tumour types e.g. from responders vs. non-responders of tumours other than melanoma?
- Fig. S12: It was concerning how similar bacterial loads were detected in both tumor and normal adjacent tissue (NAT). How different are the tumor vs. the NAT microbiome if this is the case?
