---

published: true
title: Greengenes2 unifies microbial data in a single reference tree
post_snippet: ... We introduce Greengenes2, a reference tree that unified genomic and 16S rRNA databases in a consistent, integrated resource. ... data generated from the same samples agree in principal coordinates space, taxonomy and phenotype effect size ...

---

[Rahman, Gibraan, et al. "BIRDMAn: A Bayesian differential abundance framework that enables robust inference of host-microbe associations." bioRxiv (2023).](https://www.biorxiv.org/content/10.1101/2023.01.30.526328v1.article-info) 

_This is a summary of our March 7th, 2024 DalMUG journal club discussion, written by Monica Alvaro Fuss_

### Summary

BIRDMAn (Bayesian Inferential Regression for Differential Microbiome Analysis) is a novel microbiome analysis tool for calculating differential abundance within a Bayesian framework. Quantifying differential abundance can be challenging due to the inherent characteristics of microbiome data and complex study designs, and different differential abundance tools are known to report condradictory results among biologically distinct sampling groups. Bayesian inference allows for the incorporation of prior information, enables parameter uncertainty estimation, native hierarchical modelling and edge case smoothing, which the authors aim to harness to provide customizable modelling options, as well as high confidence and reproducible results. The authors use both real and simulated data to compare BIRDMAn to existing differential abundance methods, stating that BIRDMAn provides a >20-fold immprovement in statistical power in both cross-section and longitudinal settings, capturing missed biological signals and mitigating batch effects. 

Below are the key points that came up during our discussion.

### Points of discussion

- In the simulation study, the BIRDMAn model includes a batch effect in the models for both the mean and the dispersion parameter. The other models do not appear to include a batch effect for either. And while batch specific dispersion parameters are not possible in the other packages, it would have been straightforward to include a batch-specific mean throughout.

- In the antibiotics case study, log-fold changes (b1) are allowed to vary by subject in the BIRDMAn model, whereas in the others they are not. While the other models canâ€™t include random effects, they could accommodate an ANCOVA-like formula that would allow group-specific log-fold changes.

- The cancer case study has the same issue: the BIRDMAn model allows log-fold changes to vary by group whereas the other models do not (even though they could).
