---

published: true
title: A gut microbial signature for combination immune checkpoint blockade across cancer types
post_snippet: ... strain-resolved microbial abundances improve machine learning predictions of ICB response and 12-month progression-free survival ... development of gut microbiome diagnostics or therapeutics should be tailored according to ICB treatment regimen ...

---

[Gunjur, A., Shao, Y., Rozday, T. et al. A gut microbial signature for combination immune checkpoint blockade across cancer types. Nat Med (2024). https://doi.org/10.1038/s41591-024-02823-z](https://www.nature.com/articles/s41591-024-02823-z#citeas)

_This is a summary of our March 21st, 2024 DalMUG journal club discussion, written by Monica Alvaro Fuss_

### Summary

This study examines a strain-level gut microbiome signatures associated with immune checkpoint blockade (ICB) treatment response and its ability to predict both repsonse and progression-free survival. Evidence suggests that there is a relationship between a patient's gut microbiome composition and clinical response to ICB, but generalizing microbiome-based biomarkers across cohorts remains challenging. The authors performed deep shotgun metagenomic sequencing data on 106 fecal samples from patients with one of three different types of confirmed advanced rare solid tumours and to build a custom strain-specific reference genome. The authors report differences in alpha and beta diversity between response indexes, and identified 7 strains associated with a positive response and 15 strains associated with a negative response. They state this microbial signature generalizes across the three cancer types and across different cohorts around the world, but that they do not generalize well between different treatment approaches.

Below are the key points that came up during our discussion.

### Points of discussion

- We discussed the importance of predicting ICB treatment response in cancer patients.

- We dicussed the clinical variables and why they did not contribute to the model, as well as why common measures such as PD-1 levels and tumour stage were not included. We also discussed the potential of alpha diversity measures and higher-level taxonomic groups as useful predictors. 

- We discussed the use of strain-level microbiome analysis: taxonomic resolution is important for determining functional capacity, but may hamper the identification of generalizable microbial signatures.

- We overall enjoyed the paper. We liked the machine learning workflow figure, as well as the cohort descriptions for the meta-analysis. We noted that the worst-performing cohort was also the one that used different DNA extraction kits. 
